sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/MobileCenter.java

```diff
@@ -491,8 +491,10 @@ private synchronized void setInstanceEnabled(boolean enabled) {
             mUncaughtExceptionHandler.unregister();
         }
 
-        /* Update state. */
```

Comment:
Because service checks isInstanceEnabled which an API that depends on core.isEnabled